internal/web3ext/web3ext.go

```diff
@@ -149,6 +149,11 @@ web3._extend({
 			call: 'bor_getCurrentValidators',
 			params: 0
 		}),
+		new web3._extend.Method({
+			name: 'getRootHash',
+			call: 'bor_getRootHash',
+			params: 2,
+		}),
```

Comment:
Will this be also available over WebSocket?

Reply:
Added a method in ethApi, so now it is. Checkout the PR description.